### PR TITLE
Fix resizing of debug pane in firefox

### LINKED ISSUE #348: Resizing of debug pane
resize

This bug persists in Firefox only.

### PR DESCRIPTION
**Pull request by algomaster99**
resize

Fixes #348

### CODE REVIEW

**Comment by domoritz** [comment]
Let's not make the minimum height 0 to avoid hiding the panel completely. Maybe we can limit it to 100 pixels for example.

**Comment by algomaster99** [comment]
@domoritz The input panel pane also has the `min-height` 0. Shall I make it 100 too?

**Comment by domoritz** [comment]
Probably. Do we have a constant for it? If not, let's make one.

File: src/constants/consts.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const LAYOUT = {
   DebugPaneSize: 200,
   HeaderHeight: 60,
   MinPaneSize: 30,
+  MinUpperPaneHeight: 70,
```
Comment (domoritz) [inline src/constants/consts.ts]: Let's just use `MinPaneSize`.

**Pull request merged**